homework-2/scss/style.scss

```diff
@@ -0,0 +1,1385 @@
+@import 'vars';
+@import 'fonts';
+@import 'reset';
+@import 'global';
+@import 'mixins';
+
+body.lock{
+  overflow: hidden;
+}
+.btn{
+  width: 100%;
+  padding: 8px 0; 
+  border-radius: $button-border-radius;
+  border: 1px solid $button-border-color;
+  color: $color-button;   
+  font-family: $font-links, sans-serif;
+  font-size: 10px; 
+  line-height: 24px;
+  letter-spacing: 2px;
+  text-align: center;
+  text-transform: uppercase; 
+}
+.title{
+  font-size: 24px;
+  line-height: 36px;
+}
+.text{
+  font-size: 10px;
+  line-height: 18px;
+}
+.header{
```

Comment:
Better to split code to several files. Create `_header.scss` and import it here. Same for other code.

Reply:
done